.github/workflows/sycl-linux-precommit.yml

```diff
@@ -83,7 +83,8 @@ jobs:
     name: Read compatibility testing exclude list
     runs-on: [Linux, build]
     outputs:
-      FILTER: ${{ steps.result.outputs.FILTER }}
+      FILTER_6_2: ${{ steps.result.outputs.FILTER_6_2 }}
```

Comment:
any way we could call this from a matrix so dont ned to add a new variable for every release?

Reply:
got it, thanks